src/main/webapp/app/tasks/task-list/task-list.component.spec.ts

```diff
@@ -16,7 +17,8 @@ describe('TaskListComponent', () => {
       providers: [{
         provide: 'TaskService',
         useValue: taskService
-      }]
+      }],
+      schemas: [CUSTOM_ELEMENTS_SCHEMA]
```

Comment:
I guess I was getting an error because of using angular material components. So, I added it while developing and I believe it’s not needed anymore. :) Thanks for noting that out.